src/transaction.js

```diff
@@ -288,13 +319,27 @@ export class Transaction {
           new xdr.TransactionV1Envelope({ tx, signatures })
         );
         break;
+      case xdr.EnvelopeType.envelopeTypeTxFeeBump():
+        envelope = new xdr.TransactionEnvelope.envelopeTypeTxFeeBump(
+          new xdr.FeeBumpTransactionEnvelope({ tx, signatures })
+        );
+        break;
       default:
-        throw new Error('Invalid transaction');
+        throw new Error('Unknown TransactionEnvelope');
     }
 
     return envelope;
   }
 
+  /**
+   * isFeeBump returns true if the transaction represents a FeeBumpTransaction
+   * @returns {boolean}
+   * @ignore tell jsdoc to not show this method for now
```

Comment:
Add TODO to remove ignore in next SDK release

Reply:
Done in #323